### PR TITLE
Markdown tables

### PR DESCRIPTION
**Pull request by BenGardiner**

### CODE REVIEW

**Comment by konklone** [comment]
Hey @BenGardiner, this looks like a great addition. Mind doing two things?

- [ ] Fixing the [Travis build](https://travis-ci.org/dhs-ncats/pshtt/builds/192839167), which is broken because of some `flake8` errors.
- [ ] Posting example output here in a GitHub comment (since GitHub supports Markdown and all).

File: pshtt/pshtt.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ def basic_check(endpoint):
 
     except requests.exceptions.ConnectionError:
         endpoint.live = False
-        logging.warn("Failed to connect.")
+        logging.debug("Failed to connect.")
```
Comment (konklone) [inline pshtt/pshtt.py]: :+1: from me on this. cc @h-m-f-t

**Comment by BenGardiner** [comment]
Hi,
Yes I can definitely do this and I haven't forgotten. Just haven't carved out the time yet, sorry.

**Comment by BenGardiner** [comment]
here's a sample:

|Domain |Base Domain|   Canonical URL   |Live|Redirect|Redirect To|Valid HTTPS|Defaults to HTTPS|Downgrades HTTPS|Strictly Forces HTTPS|HTTPS Bad Chain|HTTPS Bad Hostname|HTTPS Expired Cert|HSTS |HSTS Header|HSTS Max Age|HSTS Entire Domain|HSTS Preload Ready|HSTS Preload Pending|HSTS Preloaded|Domain Supports HTTPS|Domain Enforces HTTPS|Domain Uses Strong HSTS|
|-------|-----------|-------------------|----|--------|-----------|-----------|-----------------|----------------|---------------------|---------------|------------------|------------------|-----|-----------|------------|------------------|------------------|--------------------|--------------|---------------------|---------------------|-----------------------|
|dhs.gov|dhs.gov    |https://www.dhs.gov|True|False   |           |True       |True             |False           |False                |False          |False             |False             |False|           |            |False             |False             |False               |False         |True                 |True                 |False                  |

**Comment by konklone** [comment]
@h-m-f-t Thoughts?

**Comment by h-m-f-t** [comment]
@BenGardiner ...I could have _sworn_ I had already commented on this and was awaiting your response. Sorry for the delay.

It looks like the Markdown output is throwing multiple fields into one. In `Canonical URL`, it shows `https://www.dhs.gov|True|False`, leaving `Domain Enforces HTTPS` and `Domain Uses Strong HSTS` empty. 

Would you mind checking this? Your contribution is welcome!

**Comment by BenGardiner** [comment]
hmmm, yeah -- good catch.

I can definitely take a look at that.

**Comment by BenGardiner** [comment]
updated:

| Domain |Base Domain|   Canonical URL    |Live |Redirect|Redirect To|Valid HTTPS|Defaults to HTTPS|Downgrades HTTPS|Strictly Forces HTTPS|HTTPS Bad Chain|HTTPS Bad Hostname|HTTPS Expired Cert|HSTS |            HSTS Header             |HSTS Max Age|HSTS Entire Domain|HSTS Preload Ready|HSTS Preload Pending|HSTS Preloaded|Domain Supports HTTPS|Domain Enforces HTTPS|Domain Uses Strong HSTS|
|--------|-----------|--------------------|-----|--------|-----------|-----------|-----------------|----------------|---------------------|---------------|------------------|------------------|-----|------------------------------------|-----------:|------------------|------------------|--------------------|--------------|---------------------|---------------------|-----------------------|
| dhs.gov| dhs.gov   | https://www.dhs.gov| True| False  | None      | True      | True            | False          | False               | False         | False            | False            | True| max-age=31536001; includeSubDomains|    31536001| False            | False            | False              | False        | True                | True                | True                  |

**Comment by h-m-f-t** [comment]
👍  @BenGardiner! Thanks.

**Pull request merged**